src/setup/rgw.py

```diff
@@ -446,6 +446,10 @@ def _generate_ssl_cert(conf: MappedConf):
     def _update_hax_endpoint(conf: MappedConf, data_pod_hostname: str):
         """Update hax endpoint values in rgw config file."""
         Log.info('Reading motr_ha_endpoint from data pod')
+
+        if not data_pod_hostname:
```

Comment:
_update_hax_endpoint_and_create_admin() should catch this exception and avoid using it further operation, basically skip that iteration

Reply:
Done..